tests/manage/ocs_288.py

```diff
@@ -296,23 +300,22 @@ def get_pool_get_sc_name():
 
 def get_storageclass():
     """
-    Funtion for getting all storageclass
-    :return: storageclass list
+    Function for getting all storageclass
+    :return:
     """
     sc_obj = SC.get()
     sample = sc_obj['items']
+
     ignore_sc = 'gp2'
-    for item in sample:
-        if item.metadata.name not in ignore_sc:
-            storageclass_list.append(item.metadata.name)
-    if len(storageclass_list) >= 1:
-        return True
-    else:
-        log.error("No StorageClass Found")
-        return False
+    storageclass = [
+        item.metadata.name for item in sample if (
+            item.metadata.name not in ignore_sc
+        )
+    ]
+    return storageclass
 
 
-def create_secret(file_name):
+def create_secret(self, file_name):
```

Comment:
this is not good.
As this function is not a class method, it is very bad to use self here.

Reply:
Removed self in the latest commit